tests/basic.test.tsx

```diff
@@ -7,16 +7,16 @@ import {
 } from 'react'
 import { act, fireEvent, render, waitFor } from '@testing-library/react'
 import ReactDOM from 'react-dom'
-import create, { EqualityChecker, SetState, StateSelector } from 'zustand'
+import { createWithState, Store } from 'zustand'
```

Comment:
Please don't change test files, unless we really need to fix them.

Reply:
These are breaking changes. Users will have to migrate from `create` to `createWithState`, or explicitly type mutations in `create` as I mentioned [here](https://github.com/pmndrs/zustand/pull/662#discussion_r774789318). And also as per #715 (which is not finalized yet) we'd be removing those types.